pennylane_lightning/src/simulators/lightning_kokkos/gates/GatesHost.hpp

```diff
@@ -0,0 +1,1346 @@
+// Copyright 2018-2022 Xanadu Quantum Technologies Inc.
```

Comment:
Are these in addition to the LQ gates? I assume we can likely define this once and template them based on the complex type?

Reply:
Yes, but ... LQubit is only using `getPhaseShift`, `getRot`, `getRZ` and LKokkos `getHadamard`, `getRZ`, `getPhaseShift`, `getRot` (and only for tests). I think LKokkos' module is a superset of LQubit. We could get rid of the modules, or make more use of them.